src/vmm/src/vstate/vm.rs

```diff
@@ -32,7 +32,7 @@ pub enum VmError {
     /// The host kernel reports an invalid KVM API version.
     #[error("The host kernel reports an invalid KVM API version: {0}")]
     ApiVersion(i32),
-    /// Cannot initialize the KVM context due to missing capabilities.
+    /// Missing KVM capabilities.
```

Comment:
don't we want to use displaydoc for this enum?

Reply:
I do not know if `displaydoc` can be used with the complexity that is used for the `VmError::Kvm` variant so I have not changed this.